src/components/flow/routers/random/__snapshots__/RandomRouterForm.test.tsx.snap

```diff
@@ -679,102 +553,11 @@ exports[`RandomRouterForm should remove exits when shrinking 1`] = `
             >
               <temba-select
                 data-testid="temba_select_type"
+                name="type"
+                options="[{\\"name\\":\\"Send the contact a message\\",\\"value\\":\\"send_msg\\"},{\\"name\\":\\"Wait for the contact to respond\\",\\"value\\":\\"wait_for_response\\"},{\\"name\\":\\"Send somebody else a message\\",\\"value\\":\\"send_broadcast\\"},{\\"name\\":\\"Label the incoming message\\",\\"value\\":\\"add_input_labels\\"},{\\"name\\":\\"Add a URN for the contact\\",\\"value\\":\\"add_contact_urn\\"},{\\"name\\":\\"Add the contact to a group\\",\\"value\\":\\"add_contact_groups\\"},{\\"name\\":\\"Remove the contact from a group\\",\\"value\\":\\"remove_contact_groups\\"},{\\"name\\":\\"Update the contact\\",\\"value\\":\\"set_contact_field\\"},{\\"name\\":\\"Send an email\\",\\"value\\":\\"send_email\\"},{\\"name\\":\\"Save a result for this flow\\",\\"value\\":\\"set_run_result\\"},{\\"name\\":\\"Call a webhook\\",\\"value\\":\\"call_webhook\\"},{\\"name\\":\\"Call Zapier\\",\\"value\\":\\"call_resthook\\"},{\\"name\\":\\"Enter another flow\\",\\"value\\":\\"enter_flow\\"},{\\"name\\":\\"Start somebody else in a flow\\",\\"value\\":\\"start_session\\"},{\\"name\\":\\"Open a ticket with a human agent\\",\\"value\\":\\"open_ticket\\"},{\\"name\\":\\"Send the contact airtime\\",\\"value\\":\\"transfer_airtime\\"},{\\"name\\":\\"Split by intent\\",\\"value\\":\\"split_by_intent\\"},{\\"name\\":\\"Split by a custom expression\\",\\"value\\":\\"split_by_expression\\"},{\\"name\\":\\"Split by a contact field\\",\\"value\\":\\"split_by_contact_field\\"},{\\"name\\":\\"Split by a result in the flow\\",\\"value\\":\\"split_by_run_result\\"},{\\"name\\":\\"Split by random chance\\",\\"value\\":\\"split_by_random\\"},{\\"name\\":\\"Split by group membership\\",\\"value\\":\\"split_by_groups\\"},{\\"name\\":\\"Split by URN type\\",\\"value\\":\\"split_by_scheme\\"}]"
```

Comment:
Just lurking, but is this the standard way web components represent this? Feels weird having escaped JSON in an HTML attribute.